Inspicio/Inspicio.csproj

```diff
@@ -20,12 +20,26 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Content Include="wwwroot\css\home.css" />
     <Content Include="wwwroot\css\JavaScript.js" />
     <Content Include="wwwroot\css\thumbs.js" />
+    <Content Include="wwwroot\images\add.svg" />
+    <Content Include="wwwroot\images\create.svg" />
+    <Content Include="wwwroot\images\eye.jpg" />
+    <Content Include="wwwroot\images\Inspicio bar.png" />
```

Comment:
Ah that file shouldn't even be in there! Thanks! I'll get deleting the random pictures (including Newcastle.jpg one haha!)